src/pages/resultsView/enrichments/AlterationEnrichmentsTable.tsx

```diff
@@ -137,6 +137,20 @@ export default class AlterationEnrichmentTable extends React.Component<IAlterati
         return columns;
     }
 
+    @observable.ref alterationEnrichmentTableComponent:AlterationEnrichmentTableComponent|null = null;
+
+    @autobind
+    private alterationEnrichmentTableComponentRef(ref:AlterationEnrichmentTableComponent|null) {
+        this.alterationEnrichmentTableComponent = ref;
+    }
+
+    @computed get visibleGeneSymbols() {
+        if (this.alterationEnrichmentTableComponent) {
```

Comment:
seems slighty weird.  you are consulting the view component for it's state.   But the view component itself seems to update store when clicked.  So shouldn't the state (visible genes) be derivable from the store?

Reply:
see comment below.